packages/config/config/devices/0x0438/4512757.json

```diff
@@ -0,0 +1,534 @@
+{
+	"manufacturer": "Namron",
+	"manufacturerId": "0x0438",
+	"label": "4512757",
+	"description": "16A zwave thermostat - White",
+	"devices": [
+		{
+			"productType": "0x0004",
+			"productId": "0x0108"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "4",
+			"label": "ECO Mode",
+			"description": "Allowable range: 0-1",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "OFF",
+					"value": 0
+				},
+				{
+					"label": "ON",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "8",
+			"label": "Window check",
+			"description": "Allowable range: 0-1",
+			"valueSize": 4,
+			"defaultValue": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "OFF",
+					"value": 0
+				},
+				{
+					"label": "ON",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "10",
+			"label": "Frost",
+			"description": "Allowable range: 0-1",
+			"valueSize": 4,
+			"defaultValue": 0,
+			"minValue": 0,
+			"maxValue": 1,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "OFF",
+					"value": 0
+				},
+				{
+					"label": "ON",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "23",
+			"label": "Temp Unit",
+			"description": "Allowable range: 0-1",
+			"valueSize": 4,
+			"unit": "°C",
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "°C",
+					"value": 0
+				},
+				{
+					"label": "°F",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "27",
+			"label": "Temp correction",
+			"description": "Allowable range: -10~10",
+			"valueSize": 4,
+			"minValue": -10,
+			"maxValue": 10,
+			"defaultValue": 0,
+			"unsigned": false
+		},
+		{
+			"#": "31",
+			"label": "Work days set",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "5_2",
+					"value": 0
+				},
+				{
+					"label": "6_1",
+					"value": 1
+				},
+				{
+					"label": "7_0",
+					"value": 2
+				},
+				{
+					"label": "0_7",
+					"value": 3
+				}
+			]
+		},
+		{
+			"#": "40",
+			"label": "Child lock",
+			"description": "Allowable range: 0-1",
+			"valueSize": 4,
+			"defaultValue": 0,
+			"minValue": 0,
+			"maxValue": 1,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "OFF",
+					"value": 0
+				},
+				{
+					"label": "ON",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "43",
+			"label": "Sensor mode",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 6,
+			"defaultValue": 1,
+			"unsigned": true,
+			"options": [
```

Comment:
Not sure if intended, but there is a gap for value 5. If it is and only the specified options should be available to the user, you need to add
```
"allowManualEntry": false,
```
This also makes minValue and maxValue unnecessary.

Reply:
Yes there is a gap for 5 in the documentation. I have added allowManualEntry throughout where it fits.